tests/datasets/test_benin_cashews.py

```diff
@@ -28,7 +29,7 @@ def fetch(dataset_id: str, **kwargs: str) -> Dataset:
 
 
 class TestBeninSmallHolderCashews:
-    @pytest.fixture
+    @pytest.fixture()
```

Comment:
The `()` isn't actually needed unless you need to pass args to the function (don't need to change this, just letting you know for future reference)